gatsby-theme-landing-page/src/components/base.module.css

```diff
@@ -1,8 +1,51 @@
+/* Base styles and custom properties
+ *
+ * This file can be shadowed to customize the built-in components'
+ * fonts, colors, and spacing
+ */
 .root {
+  /* typography */
```

Comment:
I think it'd make sense to separate the custom property definitions from the base styles to make it easier to shadow one without the other